src/utils/getTodoById.ts

```
@@ -0,0 +1,5 @@
+import { Todo } from '../types/Todo';
+
+export const getTodoById = (todos: Todo[], id: number) => {
+  return todos.find(todo => todo.id === id)!;
```

Using the non-null assertion operator (`!`) assumes that a todo with the given ID will always be found. Consider handling the case where the todo is not found to avoid potential runtime errors. You could return `undefined` or throw an error if the todo is not found.